internal/objects/reconciler_test.go

```diff
@@ -467,6 +472,18 @@ var _ = Describe("Basic propagation", func() {
 		Eventually(HasObject(ctx, "configmaps", barName, "cm-with-label-2")).Should(BeTrue())
 	})
 
+	It("should not propagate objects excluded by labels when using all annotation", func() {
```

Comment:
This is a change from the current behaviour - why not allow the exclusion to be overridden?

Either way, can you please document this as well?

Reply:
Hmm the thing is that it's indeed a change in behavior, however as we discussed on the original issue, it was never intended. In addition, the current documentation never mentioned that it can work this way - it only says that the 'all' annotation is useful on the AllowPropagate mode. I actually discovered this behavior by looking at the code. What are your thoughts here? Any suggestion as to how we should document it further than what I mentioned?

Reply:
I tend to agree with @duduedri96 but do understand this is a breaking change so we may want to consider not breaking the behavior. I guess I'm saying I could go either way here :).

Reply:
Ok, I'm good to leave the (new) behaviour as-is, and yes, I don't think we need any more documentation unless someone asks for it.